### PR TITLE
[WS-38] Add a picture to a course for use it in the main page and about page

### PR DESCRIPTION
**Pull request by vladimir-seleznev**
**Description:** Add a picture to a course for use it in the main page and about page.

**Youtrack:** https://youtrack.raccoongang.com/issue/WS-38
 https://youtrack.raccoongang.com/issue/WS-33

### CODE REVIEW

File: cms/static/js/views/settings/main.js [hunk 1]
```diff
@@ -436,6 +444,12 @@ define(['js/views/validation', 'codemirror', 'underscore', 'jquery', 'jquery.ui'
                        image_key = 'web_science_image_name';
                        image_path_key = 'web_science_image';
                        break;
+                   case 'upload-web-science-main-image':
+                       title = gettext('Upload your course main image.');
```
Comment (sendr) [inline cms/static/js/views/settings/main.js]: Are you going to add appropriate translates or they have been added before?

File: cms/templates/settings.html [hunk 2]
```diff
@@ -343,6 +343,33 @@ <h2 class="title-2">${_('Course Details')}</h2>
                     <button type="button" class="action action-upload-image" id="upload-web-science-image">${_("Upload Web Science Course Card Image")}</button>
                   </div>
                 </li>
+                <li class="field image" id="field-web-science-main-image">
+                  <label for="course-web-science-url">${_("Web Science course main Image")}</label>
```
Comment (sendr) [inline cms/templates/settings.html]: the same question about translates. And maybe better to use `platform_name` instead of `Web Science`?

Comment (vladimir-seleznev) [reply cms/templates/settings.html]: I'll change `Web Science` to `platform name`

File: cms/static/js/views/settings/main.js (hunk 1 above)
Comment (vladimir-seleznev) [reply cms/static/js/views/settings/main.js]: if we need a translation, we could use it in future, like another's gettext in this template.

File: cms/templates/settings.html (hunk 2 above)
Comment (vladimir-seleznev) [reply cms/templates/settings.html]: done

**Pull request merged**